src/applications/claims-status/components/ClaimDetailLayout.jsx

```diff
@@ -11,7 +11,7 @@ import ClaimsBreadcrumbs from './ClaimsBreadcrumbs';
 import ClaimsUnavailable from './ClaimsUnavailable';
 import { isPopulatedClaim, getClaimType } from '../utils/helpers';
 
-const MAX_CONTENTIONS = 3;
+const MAX_CONTENTIONS = 10;
```

Comment:
I'm not sure that this is the way that we want to accomplish our goal here. This is what a real claim can look like: 
<img width="437" alt="Screenshot 2023-11-16 at 2 03 27 PM" src="https://github.com/department-of-veterans-affairs/vets-website/assets/13838621/a9d8e897-9ac7-40ab-a5b9-b893cc6b2702">

Reply:
I think we may just want to leave this at 3